storages/backends/gcloud.py

```diff
@@ -20,6 +21,7 @@
 class GoogleCloudFile(File):
     def __init__(self, name, mode, storage):
         self.name = name
+        self.mime_type = mimetypes.guess_type(name)[0]
```

Comment:
Since `guess_type` isn't 100% reliable, I think we need to allow the user to override the type here if needed. Can you add that?